tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/bluetooth.py

```diff
@@ -4,55 +4,61 @@
 
 from marionette_driver import expected, By, Wait
 
-from gaiatest.apps.base import PageRegion
+from gaiatest.apps.base import Base
```

Comment:
I don't really understand why you're moving away from PageRegion here. Could you explain?